static/app/views/settings/project/server-side-sampling/samplingSDKAlert.tsx

```diff
@@ -67,8 +66,10 @@ export function SamplingSDKAlert({
         ? t(
             'Server-side sampling rules are in effect without the following SDK’s being updated to their latest version.'
           )
-        : t(
-            'To keep a consistent amount of transactions across your applications multiple services, we recommend you update the SDK versions for the following projects:'
+        : tn(
+            'To keep a consistent amount of transactions across your applications multiple services, we recommend you update the SDK versions for the following project:',
+            'To keep a consistent amount of transactions across your applications multiple services, we recommend you update the SDK versions for the following projects:',
```

Comment:
i was wondering to copy string to variable and use it in both cases: `project` and `projects`

Reply:
I understand, but when it comes to translations this is a normal pattern and I would rather not change that

Reply:
I'm afraid that when we keep just parts of this translation elsewhere, it won't make much sense when translating to other languages